eve/main.yml

```diff
@@ -88,6 +93,12 @@ stages:
           env:
             <<: *deploy-env
             <<: *keycloak-env
+      - ShellCommand:
+          name: upload test coverage
+          haltOnFailure: False
```

Comment:
Just a suggestion, the default is already `haltOnFailure: False` so we could remove this line.

But sometimes you do want to make this explicit so I'll leave it up to you.
```suggestion
```

Reply:
I was following the test suite that includes it, I can remove both though as it is the default.